Dockerfile

```diff
@@ -1,9 +1,27 @@
+##############################
+## Compile dist CSS
+##############################
+FROM node:12-alpine AS compile_css
```

Comment:
Maybe use newer Node? Node 12 has EOL on 30 Apr 2022 for security updates (bug fixes ended on 20 Oct 2020).

Reply:
Yeah I had 16 originally but that wouldn't run. Downgrading fixed it. Can maybe downgrade a little less aggressively though.

Reply:
Ok, checked my local and I have version 15. Updated the Dockerfile to 15 and that worked fine. Also dropped the `apk` install line below which was part of my trial-and-error approach to fixing the problem and turns out to not be needed..